setup.py

```diff
@@ -18,5 +18,8 @@
           "Programming Language :: Python :: 2",
           "Programming Language :: Python :: 3",
           "Topic :: Scientific/Engineering",
+      ],
+      data_files=[
```

Comment:
You should have a look at [this lines](https://github.com/astralien3000/ikpy/blob/dev-astra2/src/ikpy/blender_utils.py#L39) to understand the dependency of the library on the script.


To sum it up, it is almost impossible to parse directly the .blend file (because it is a binary format, which can potentially change with any version of blender). So, I use blender itself and it's python API to get the information I need.

This makes me think the way I did that is not really portable. I did not test for Windows...